documentation/compatibility_with_puppet_agent.markdown

```diff
@@ -0,0 +1,106 @@
+# Puppet 3 Backwards Compatibility
+
+Puppet Server 2.1 introduces a new feature that allows Puppet 3 agents to
+operate with puppetserver.  Puppet Server 1.x requires Puppet 3 while Puppet
+Server 2.x requires Puppet 4.  These requirements created a situation where
+only Puppet 4 agents operate with Puppet Server 2.0 masters.  We've added
+Puppet 3 support in Puppet Server 2.1 to ease the upgrade process for users.
+
+Compatibility is provided by a so-called `legacy-routes-service`.  This service
```

Comment:
Heh, "so-called" sounds a little funny... but I'm guessing the docs team is going to punch all this stuff up?